Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -86,6 +88,11 @@ class TabManagerMiddleware {
             store.dispatch(TabPanelAction.refreshTab(tabs))
             store.dispatch(TabTrayAction.dismissTabTray)
 
+        case RemoteTabsPanelAction.openSelectedURL(let url):
```

Comment:
This is new behavior that defers from the Legacy version. We will open a new tab using the URL selected from Remote tabs instead of opening the URL in the selected tabs as the Legacy version. 

Confirmed with @dnarcese this is the wanted behaviour for the new Tab Tray